test/config/utility.cc

```diff
@@ -0,0 +1,98 @@
+#include "test/config/utility.h"
+
+#include "common/common/assert.h"
+#include "common/protobuf/utility.h"
+
+#include "test/test_common/environment.h"
+#include "test/test_common/network_utility.h"
+
+#include "api/filter/http_connection_manager.pb.h"
+
+namespace Envoy {
+
+std::string basic_config = R"EOF(
+{
+  "static_resources": {
```

Comment:
Thoughts on using YAML for this so we can put comments directly into the config snippets that we will end up having? Not sure if worth it or not. (No real preference).

Reply:
Let's see what the config snippets end up being and then I can always change over if we think it's worth it, SG?

Reply:
actually, @htuch want to weigh in here as well?

Reply:
If humans are to consume/maintain, YAML is always best IMHO; JSON is just so much more verbose and noisy.

Reply:
Alas for my laziness, YAML it is then! =P
Will upload a new version, likely Monday.